CONTRIBUTING.md

```diff
@@ -21,6 +21,17 @@ To contribute to this project, it is recommended that you follow these steps:
 
 1. Fork the repository on GitHub.
 2. Build. Unit tests will execute as part of this.
+   - To build on a Windows Machine
+      1. Install Python from https://www.python.org/downloads/ . 
+      2. Add Python install location to the Windows path.
+      3. Install PyPI, following the instructions on https://pip.pypa.io/en/latest/installing/ .
+      4. Install Wheel using PyPI.
```

Comment:
Group all the python modules.

4. Install required Python modules using PyPI
   * wheel
   * foo
   * bad